common/sagas/wallet/wallet.ts

```diff
@@ -141,18 +146,43 @@ export function* unlockPrivateKey(action: UnlockPrivateKeyAction): SagaIterator
   yield put(setWallet(wallet));
 }
 
+export function* startLoadingSpinner(): SagaIterator {
+  yield call(delay, 1000);
```

Comment:
I'd probably shorten this to 300-500, a full second before having the site respond is a little long.